tabbycat/draw/generator/random.py

```diff
@@ -2,7 +2,7 @@
 
 import random
 
-from django.utils.translation import gettext as _
+from django.utils.translation import gettext
```

Comment:
Importing this as an underscore is standard in internationalization: 
https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#internationalization-in-python-code
https://docs.python.org/3/library/gettext.html#gettext.gettext

One can argue about whether this is a good thing, but it's the standard we've chosen to follow, since we translate a lot of strings and this is the most common way of avoiding clutter with `gettext`.

Reply:
Ok. Seems a bit strange, however.